src/App.js

```diff
@@ -1,14 +1,26 @@
 import React from 'react';
+import './App.css';
+import { Route, Switch } from 'react-router-dom';
+import Navbar from './components/Navbar';
 import Rockets from './components/Rockets';
 import Missions from './components/Missions';
 import Profile from './components/Profile';
 
 export default function App() {
   return (
     <div>
-      <Rockets />
-      <Missions />
-      <Profile />
+      <Navbar />
+      <Switch>
+        <Route exact path="/">
+          <Rockets />
+        </Route>
+        <Route path="/missions">
+          <Missions />
+        </Route>
+        <Route path="/myprofile">
+          <Profile />
+        </Route>
```

Comment:
- [ ] All component children of Routes must be a Route. So a better way of writing this after changing the Switch to Routes is <Route exact path="/" element={ < Profiles /> } />